Doc/library/datetime.rst

```diff
@@ -283,15 +283,13 @@ Supported operations:
 | ``abs(t)``                     | equivalent to +\ *t* when ``t.days >= 0``, and|
 |                                | to -*t* when ``t.days < 0``. (2)              |
 +--------------------------------+-----------------------------------------------+
-| ``str(t)``                     | Returns a string in the form                  |
-|                                | ``[D day[s], ][H]H:MM:SS[.UUUUUU]``, where D  |
-|                                | is negative for negative ``t``. (5)           |
+| ``str(t)``                     | Returns a string in the form                  | |                                | ``[D day[s], ][H]H:MM:SS[.UUUUUU]``, where D  | |                                | is negative for negative ``t``. (5)           |
```

Comment:
This line seems weird. Did you try to compile the doc? make -C Doc html

Why did you touch this doc?